sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/LICtlWorkload.java

```diff
@@ -20,6 +20,14 @@
 public class LICtlWorkload {
     private static final Logger LOGGER = LoggerFactory.getLogger(LICtlWorkload.class);
 
+    /**
+     * The test scenarios supported for the LinkedIn CTL Workload
+     */
+    public enum Scenario {
+        GET,
+        QUERY
```

Comment:
It will be used. For the % based workload where we run X% of GETs and say 100-X% of QUERY, I am going to add a new param/type called 'Composite' or 'Hybrid' which runs multiple scenarios in one run.

This Scenario, passed in from the command line, is good to have. Its gives the flexibility to run a GET-only or QUERY-only scenario if needed.